packages/react/src/ml/useDismissRecommendedAccounts.ts

```diff
@@ -0,0 +1,37 @@
+import type {
+  UnauthenticatedError,
+  UnexpectedError,
+} from '@lens-protocol/client';
+import { dismissRecommendedAccounts } from '@lens-protocol/client/actions';
+
+import type { DismissRecommendedAccountsRequest } from '@lens-protocol/graphql';
+import { type UseAsyncTask, useAuthenticatedAsyncTask } from '../helpers';
+
+/**
+ * Dismiss recommended accounts.
+ *
+ * @alpha This is an alpha API and may be subject to breaking changes.
+ *
+ * ```tsx
+ * import { evmAddress, useDismissRecommendedAccounts } from '@lens-protocol/react';
+ *
+ * const { execute } = useDismissRecommendedAccounts();
+ *
+ * const result = await execute({
+ *   accounts: [evmAddress('0x1234567890123456789012345678901234567890')],
+ * });
+ *
+ * if (result.isErr()) {
+ *   console.error(result.error);
+ * }
+ * ```
+ */
+export function useDismissRecommendedAccounts(): UseAsyncTask<
+  DismissRecommendedAccountsRequest,
+  void,
+  UnauthenticatedError | UnexpectedError
+> {
+  return useAuthenticatedAsyncTask((sessionClient, request) =>
+    dismissRecommendedAccounts(sessionClient, request),
```

Comment:
Should we refetch recommended account or invalidate that query. Not sure if urql has similar concept where you invalidate the query and, if it was listened by some other part of the UI, it retriggers it. Worth checking though.